# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../embutils/no-shared.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We later might like to disable more.

  - Rene Rebe <rene@exactcode.de>

--- ./Makefile~	2006-11-05 11:51:14.000000000 +0100
+++ ./Makefile	2006-11-05 11:51:38.000000000 +0100
@@ -13,3 +13,3 @@
 sosrm soscp sosmv sosln soslns md5sum sleep2 allinone uniq tr mesg du \
-uuencode uudecode nohup nice cmp mktemp truncate strings test date \
+uuencode uudecode nohup nice cmp mktemp truncate test date \
 printenv chrootuid renice \
